# some of these headers are in the wrong order causing __packed to become a
# global variable,  the DAL -fno-common linker flag weed these issues out.

--- aircrack-ng-1.5.2/src/aircrack-osdep/radiotap/radiotap.c	2019-09-05 08:41:32.270655860 +1000
+++ aircrack-ng-1.5.2/src/aircrack-osdep/radiotap/radiotap.c	2019-09-05 08:41:38.538655996 +1000
@@ -11,8 +11,8 @@
  * Alternatively, this software may be distributed under the terms of ISC
  * license, see COPYING for more details.
  */
-#include "radiotap_iter.h"
 #include "platform.h"
+#include "radiotap_iter.h"
 
 /* function prototypes and related defs are in radiotap_iter.h */
 
--- aircrack-ng-1.5.2/src/aircrack-osdep/linux.c	2019-09-05 08:41:45.286656143 +1000
+++ aircrack-ng-1.5.2/src/aircrack-osdep/linux.c	2019-09-05 08:42:02.834656524 +1000
@@ -52,8 +52,6 @@
 #include <linux/genetlink.h>
 #endif // CONFIG_LIBNL
 
-#include "radiotap/radiotap.h"
-#include "radiotap/radiotap_iter.h"
 /* radiotap-parser defines types like u8 that
 		 * ieee80211_radiotap.h needs
 		 *
@@ -63,6 +61,8 @@
 		 * - since linux does not include it in userspace headers
 		 */
 #include "osdep.h"
+#include "radiotap/radiotap.h"
+#include "radiotap/radiotap_iter.h"
 #include "pcap.h"
 #include "crctable_osdep.h"
 #include "common.h"
